flixel/FlxObject.hx

```diff
@@ -548,16 +539,26 @@ class FlxObject extends FlxBasic
 	/**
 	 * Overriding this will force a specific color to be used for debug rect.
 	 */
-	public var debugBoundingBoxColor:Null<Int> = null;
-	
-	public var debugColorScheme(default, null):DebugColorScheme;
+	public var debugBoundingBoxColor(default, set):Null<Int> = null;
+	public var debugBoundingBoxColorNotSolid(default, set):Null<Int> = null;
+	public var debugBoundingBoxColorPartial(default, set):Null<Int> = null;	
```

Comment:
I'm not sure. I want to use `debugBoundingBoxColor` which is the original field prior to my PR. Won't changing the type cause any breaking? Or should I not mess with the original field at all and should create a new one?